src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java

```diff
@@ -119,10 +113,8 @@ public List<User> getAllUsers(){
     @Override
     public User getUserById(int id) {
 
-        String sql = "SELECT * FROM login WHERE id = ?";
-
         try (Connection con = ConnectionHelper.getConnection();
-             PreparedStatement stmt = con.prepareStatement(sql)) {
+             PreparedStatement stmt = con.prepareStatement(GET_USER_BY_ID_STATEMENT)) {
```

Comment:
fix the try-with-resource. remove preparedstatement from try()

Reply:
okay napo sir